### ISSUE TITLE
Refine guidelines for Rmarkdown files

### ISSUE THREAD

**Issue by careymcg** [afsc-assessments/funplots#3]
- [ ] should plots be saved locally, then included in Rmarkdown or included directly?
- [ ] best method for controlling figure size and font size in figures when doing Rmarkdown?
- [ ] how many .Rmd files should be made to feed into a single report and what should they look like?
- [ ] where should .Rmd files go in the package and where should resulting docs go? In a /doc folder?

**Comment by BenWilliams-NOAA** [comment]
i recommend keeping the plotting package wholly separate from rmd templates. a common method for generating rmd/report files is to store them  in a the `inst` folder as a `skeleton` see https://github.com/BenWilliams-NOAA/safe/tree/main/inst/rmarkdown/templates/safe/skeleton for an example approach. if you roll all of this together you will have a massive package and it will be hard to parse out issues.
